js/lib/d3-timeline.js

```diff
@@ -1,4 +1,7 @@
 // vim: ts=2 sw=2
+var scrolledX;
+var zoomedWidth;
```

Comment:
This can be done without a global variable (for reasons why global variables are bad: https://gist.github.com/hallettj/64478). The d3-timeline exposes functions to retrieve variables. I believe there is already a function called `width()` in d3-timeline that you can call to get the width. You might want to make a function for the `scrolledX` similar to the `width()` function.